applier/docker.go

```diff
@@ -0,0 +1,122 @@
+package applier
```

Comment:
It's nice to see we can do major refactors and re-writes like this without breaking functionality since `docker_test.go` exists! 🤞